tony-history-server/app/utils/HdfsUtils.java

```diff
@@ -63,27 +59,52 @@ static String contentOfHdfsFile(FileSystem fs, Path filePath) {
   }
 
   /**
-   * List all metadata file paths in {@code histFolder}.
+   * Extract the job id portion from the file path string.
+   * @param path file path string.
+   * @return job id
+   */
+  public static String getJobId(String path) {
```

Comment:
The name of this method could be made more general, given what it's doing. The method is just returning the last component of a path.

Reply:
I agree. But in our case, I think the `HdfsUtils` API is more intuitive and easier to use with the name `getJobId`, compared to a more generic name like `getLastComponent`.

Reply:
How about `getJobIdComponent` then?